extensions/git-commands/src/command-detail.tsx

```diff
@@ -1,36 +1,95 @@
-import { Detail } from "@raycast/api";
-import { typeColor, typeDescription } from "./utils";
+import { Action, ActionPanel, Color, Detail, Icon } from "@raycast/api";
+import { typeColor, typeDescription, mainCommand } from "./utils";
+import { useState } from "react";
 import { Alias } from "./types";
 
-export default function CommandDetail({ alias }: { alias: Alias }) {
+export default function CommandDetail({
+  alias,
+  isFavorite: _isFavorite,
+  onFavorite,
+  onCopy,
+}: {
+  alias: Alias;
+  isFavorite: boolean;
+  onFavorite: () => void;
+  onCopy: () => void;
+}) {
+  const [isFavorite, setIsFavorite] = useState(_isFavorite);
+  const { name, command, type, description } = alias;
+  const onGitName = `Command "${mainCommand(command)}" documentation`;
+  const onGitURL = `https://git-scm.com/docs/git-${mainCommand(command)}`;
+  const searchName = "Search for complete command";
+  const searchURL = `https://git-scm.com/search/results?search=${encodeURIComponent(command)}`;
+
+  const handleFavorite = () => {
+    setIsFavorite(!isFavorite);
+    onFavorite();
+  };
+
   const markdown = `
-# ${alias.name}
+# ${name}
 ###
 \`\`\`
-${alias.command}
+${command}
 \`\`\`
 
-## ${alias.description}
+### ${description}
+
+${typeDescription(type)}
+
+### Git Documentation
+- [${onGitName} ↗](${onGitURL})
+- [${searchName} ↗](${searchURL})
 ###
-\`${alias.type}\` ${typeDescription(alias.type)}
 `;
   return (
     <Detail
       markdown={markdown}
+      actions={
+        <ActionPanel>
+          <Action.CopyToClipboard title="Copy Alias" content={name} onCopy={onCopy} />
+
+          <ActionPanel.Section>
+            {isFavorite ? (
+              <Action
+                icon={Icon.StarDisabled}
+                title="Remove Favorites"
+                onAction={handleFavorite}
+                shortcut={{ modifiers: ["cmd"], key: "x" }}
+              />
+            ) : (
+              <Action
+                icon={Icon.Star}
+                title="Save Favorite"
+                onAction={handleFavorite}
+                shortcut={{ modifiers: ["cmd"], key: "s" }}
```

Comment:
```suggestion
                shortcut={Keyboard.Shortcut.Common.Pin}
```
Maybe use the default shortcut instead?